src/Sequelize/Models.js

```diff
@@ -78,3 +78,10 @@ exports._belongsToMany = function (target, source, thru) {
     return null;
   };
 };
+
+exports._belongsToWithFKey = function (target, source, alias, fKey, targetKey) {
```

Comment:
You can take a single json object as "options" and make that have { alias, fKey, targetKey}
This is extendible if you want to include other options in future. Look at the current _belongsTo function, we are making same mistake again!

Reply:
Is it okay to change the current instance of _belongsTo as we are not using it as of now in our codeBase.

Reply:
We shouldn't do that. This is our public repo. You never know who else is using this too. We shouldn't break backward compatibility.

Reply:
Ya got it